### DISCUSSION TITLE
Need version 0.9.1 of Stellarium

### DISCUSSION THREAD

**Discussion by Hondoe1950** [Stellarium/stellarium#1654] [General]
Hi! I have a program that links to Stellarium 0.9.1 and won't work with any subsequent versions.

Does anyone have a copy of this version that I can have?

I would really appreciate it.

**Comment by gzotti** [comment]
Errm, why not use the search engine of your preference?

**Comment by Hondoe1950** [comment]
> Errm, why not use the search engine of your preference?

I have used Google and spent over 40 minutes trying to find version 0.9.1 to no avail. Every link eventually pointed me to the current version

**Reply by gzotti** [reply]
Oops, I thought they were still available on SourceForge. Seems we removed the hopelessly outdated prehistoric versions. Anyhow, forget and delete everything before 0.14.2 because of accuracy issues. Ask the developer of that other software to catch up with the decade that lies between then and today.

**Comment by axd1967** [comment]
Maybe tell us what the other application is ?

**Reply by Hondoe1950** [reply]
The program is Solar Fire, the foremost astrology program. Unfortunately, the company has not responded to my requests, up to now, to make changes in it so that it will work with later versions of Stellarium.

**Comment by axd1967** [comment]
Maybe this can help https://archive.org/details/dvdrom-DPK_2008_08

# BUT...

do not complain if you get viruses on your PC/laptop/whatever! it's REALLY not worth it.

I looked into the User Manual (https://www.solarfire.info/SolarFiremanual.htm) and it no longer mentions Stellarium, so it looks like support has been dropped for good.

Why did you need Stellarium for?

**Reply by Hondoe1950** [reply]
I agree, too risky.

**Comment by axd1967** [comment]
Meanwhile I found this: http://www.soulhealing.com/TMASoftStarIssue710.htm

If all you need is a sky view of a date at a given location, you don't need any linking. Just select the correct date, time and location and you can get something like this:


![image](https://user-images.githubusercontent.com/3529789/117576159-08688900-b0e5-11eb-8b2d-7caf55e673cf.png)

![image](https://user-images.githubusercontent.com/3529789/117576177-1c13ef80-b0e5-11eb-81fb-0e5ff249d543.png)

![image](https://user-images.githubusercontent.com/3529789/117576194-30f08300-b0e5-11eb-9222-46e1880edb41.png)

![image](https://user-images.githubusercontent.com/3529789/117576235-6eeda700-b0e5-11eb-846f-4f37a377b77e.png)


etcetera

**Reply by Hondoe1950** [reply]
Thanks! Sadly, Solar Fire used to transfer the birth data automatically to Stellarium, but not since 0.9.1
